pkg/services/searchusers/sortopts/sortopts.go

```diff
@@ -0,0 +1,91 @@
+package sortopts
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/grafana/grafana/pkg/services/search/model"
+	"github.com/grafana/grafana/pkg/util/errutil"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+var (
+	// SortOptionsByQueryParam is a map to translate the "sort" query param values to SortOption(s)
+	SortOptionsByQueryParam = map[string]model.SortOption{
+		"login-asc":          newSortOption("login", false, 0),
+		"login-desc":         newSortOption("login", true, 0),
+		"email-asc":          newSortOption("email", false, 1),
+		"email-desc":         newSortOption("email", true, 1),
+		"name-asc":           newSortOption("name", false, 2),
+		"name-desc":          newSortOption("name", true, 2),
+		"lastSeenAtAge-asc":  newTimeSortOption("last_seen_at", false, 3),
+		"lastSeenAtAge-desc": newTimeSortOption("last_seen_at", true, 3),
+	}
+
+	ErrorUnknownSortingOption = errutil.BadRequest("unknown sorting option")
```

Comment:
@linoman here is the definition of the error :)

Reply:
Much cleaner!